Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -38,6 +39,7 @@ public class BacktestingBrokerage : Brokerage
         private readonly ConcurrentDictionary<int, Order> _pending;
         private readonly object _needsScanLock = new object();
         private readonly HashSet<Symbol> _pendingOptionAssignments = new HashSet<Symbol>();
+        private readonly Dictionary<TimeInForce, ITimeInForceHandler> _timeInForceHandlers = new Dictionary<TimeInForce, ITimeInForceHandler>();
```

Comment:
Hopefully we can change the implementations of `ITimeInForceHandler` such that they pull all information from the method parameters leading to a parameterless constructor, and then we can inline the creating of this map here of the form:
``` csharp
private readonly Dictionary<TimeInForce, ITimeInForceHandler> _timeInForceHandlers = new Dictionary<TimeInForce, ITimeInForceHandler>
{
    { TimeInForce.GoodTilCanceled, new GoodTilCanceledTimeInForceHandler() }
};
```